### PR TITLE
Coverity ID 1241990: Fixes memory leak

### PR DESCRIPTION
**Pull request by zwoop**
Susan: I need some input here if this is as expected, it seems odd that it could (before) possibly insert an empty (nullptr) entry, right ?

### CODE REVIEW

File: plugins/experimental/ssl_cert_loader/ssl-cert-loader.cc [hunk 1]
```diff
@@ -297,32 +296,40 @@ Parse_Config(Value &parent, ParsedSslValues &orig_values)
...
-    if (entry != nullptr) {
+      if (cur_values.server_ips.size() > 0) {
+        for (auto &server_ip : cur_values.server_ips) {
+          IpEndpoint first, second;
+          first.assign(server_ip.first);
+          second.assign(server_ip.second);
+          Lookup.ipmap.fill(&first, &second, entry);
+          char val1[256], val2[256];
+          server_ip.first.toString(val1, sizeof(val1));
+          server_ip.second.toString(val2, sizeof(val2));
```
Comment (ywkaras) [inline plugins/experimental/ssl_cert_loader/ssl-cert-loader.cc]: What purpose does it serve to fill in the block-local strings val1 and val2?

Comment (shinrich) [reply plugins/experimental/ssl_cert_loader/ssl-cert-loader.cc]: Probably remnants of past debugging efforts.

**Comment by shinrich** [comment]
@zwoop are you asking about adding the check for entry != nullptr before inserting?  That seems perfectly reasonable.  The code change looks good to me.

**Comment by zwoop** [comment]
@shinrich Correct. And thanks for the review!

File: plugins/experimental/ssl_cert_loader/ssl-cert-loader.cc (hunk 1 above)
Comment (zwoop) [reply plugins/experimental/ssl_cert_loader/ssl-cert-loader.cc]: Yeh, I didn't look at any of that, just trying to address the coverity memory leak issue :).

**Pull request merged**